packages/story-editor/src/app/rightClickMenu/constants.js

```diff
@@ -32,11 +32,17 @@ export const RIGHT_CLICK_MENU_LABELS = {
   ADD_TO_TEXT_PRESETS: __('Add Style to “Saved Styles”', 'web-stories'),
   BRING_FORWARD: __('Bring Forward', 'web-stories'),
   BRING_TO_FRONT: __('Bring to Front', 'web-stories'),
+  BROWSE_STOCK_IMAGES_AND_VIDEO: __(
+    'Browse Stock Images and Video',
+    'web-stories'
+  ),
```

Comment:
It bugs me that this menu item line-breaks and the others don't. I do know that's a very English-centric viewpoint and it might very well line-break (or not) in other languages too, but could we make the popup at least wide enough to fit all the buttons on one line in English (as they do in Figma)?

| Figma | Editor |
|-|-|
| ![Screen Shot 2022-01-19 at 17 56 49](https://user-images.githubusercontent.com/637548/150232118-3dbdbad2-4f68-4215-b967-b5694762d0b8.png) | ![Screen Shot 2022-01-19 at 17 57 01](https://user-images.githubusercontent.com/637548/150232127-49e15f49-96f0-41b0-8fe3-99ed4e8835fa.png) |

Reply:
It bugged me too until I saw that the right padding in Figma is adjusted depending on the need 😉 

![image](https://user-images.githubusercontent.com/21036224/150381733-eb438dc1-2491-4faa-97a3-473c86481138.png)

I will change the hardcoded `MenuWrapper` width to 218 👍🏻